### ISSUE TITLE
nginx-proxy forwarding to ports 8080, 8081, etc....

### ISSUE THREAD

**Issue by JulianKingman** [jwilder/nginx-proxy#266]
My set up is currently two test containers running Wordpress on different subdomains. It's appending 8080 and 8081 (8082 would be next, I assume) to them for some reason. Any idea why this is happening? If I explicitly point 8080 to port 80 in the container, it works, but it looks like this: server1.designman.io:8080.

Nginx-proxy:
````
docker run -d -p 80:80 -v /var/run/docker.sock:/tmp/docker.sock:ro jwilder/nginx-proxy
````

Database:
````
docker run --name wp_db_1 -e MYSQL_ROOT_PASSWORD=w0mq9ONY0AWc  -d mariadb
````
Wordpress:
````
docker run -e WORDPRESS_DB_PASSWORD=w0mq9ONY0AWc -e VIRTUAL_HOST=server1.designman.io* -d --name wp_2 --link wp_db_1:mysql --expose=80 -v "$PWD/wp_1":/var/www/html  wordpress
````

**Comment by jwilder** [comment]
Where are you seeing 8080 appended?

**Comment by md5** [comment]
It sounds to me like the Wordpress database was set up when the sites were using those ports and that it's probably in the database. If that's the case, you'll need to search and replace in the database dump to get rid of those ports.

**Comment by JulianKingman** [comment]
@md5 That was a great idea, but didn't work out... To simplify, I deleted the containers and created new ones. It then occurred to me that it could be a file system thing, so I deleted the files, as well, but no luck.

It actually seems like it may be a different issue, as opening an incognito window gives me a "503 Service Temporarily Unavailable" and doesn't append the port (same as issue #157, but without the port mapping, as you can see). Here's the URL if you want to look: http://server1.designman.io/

**Comment by JulianKingman** [comment]
I'm running docker on CoreOS, would that make any difference?

**Comment by jwilder** [comment]
@JulianKingman Your `VIRTUAL_HOST` var has `*` at the end?  What happens if you remove that?

**Comment by jwilder** [comment]
Also, any errors shown in the `nginx-proxy` container logs?

**Comment by JulianKingman** [comment]
Thanks for all your speedy help @jwilder and @md5, I got it sorted out.

I think the main problem was a browser caching issue, which I figured out once I opened it in an incognito window and got the nginx error. I then realized that in tweaking various things, I had put quotes around VIRTUAL_HOST=server1.designman.io, and when I removed them, it worked like a charm! The * was part of debugging, and didn't seem to have much impact either way.
